examples/ex_rng_scope.cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+#include <igraph.hpp>
+#include "ex_vector_print.hpp"
+
+int main() {
+    // Check that we can influence the RNG.
+    {
+        ig::RNGScope filled(1000);
+        std::cout << "New RNG scope:" << std::endl;
+        std::cout << "  Draw from [0, 100]: " << igraph_rng_get_integer(igraph_rng_default(), 0, 100) << std::endl;
```

Comment:
Can you please use `RNG_INTEGER` and the similar macros in all places in the example?